demosplan/DemosPlanCoreBundle/Logic/Report/ReportService.php

```diff
@@ -74,6 +74,19 @@ function (EntityManagerInterface $em) use ($reportEntries) {
         );
     }
 
+    public function persistAndFlushWithoutTransaction(ReportEntry ...$reportEntries)
+    {
+        foreach ($reportEntries as $reportEntry) {
+            $violations = $this->validator->validate($reportEntry);
+            if (0 !== $violations->count()) {
+                throw ViolationsException::fromConstraintViolationList($violations);
+            }
+            $this->reportRepository->addObject($reportEntry);
+        }
+
+        return null;
```

Comment:
nitpick: you could make this method void

Reply:
done in [e753db9](https://github.com/demos-europe/demosplan-core/pull/3014/commits/e753db99a1aecc1a42c9037150bb18ae276363d7)